src/platform/plugins/shared/discover/public/application/main/state_management/redux/actions/tabs.ts

```diff
@@ -99,7 +103,14 @@ export const updateTabs: InternalStateThunkActionCreator<[TabbedContentState], P
       const existingTab = selectTab(currentState, item.id);
 
       const tab: TabState = {
-        ...defaultTabState,
+        ...DEFAULT_TAB_STATE,
+        ...{
+          globalState: {
+            timeRange: services.timefilter.getTime(),
+            refreshInterval: services.timefilter.getRefreshInterval(),
+            filters: services.filterManager.getGlobalFilters(),
+          },
```

Comment:
Preserving the current global state when creating a new tab since we no longer rely directly on global services.